README.md

```diff
@@ -1,3 +1,23 @@
-`pes_manifest.txt` contains manifest for Packge Evolution Service
+# PES manifest
 
-Note: When refreshing backend packages, remove kernel entry.
+```bash
+├── generate_manifest.py # Script which creates manifest from three files with packages
+├── pes_backend_packages.txt # Python packages used in PES
+├── pes_container_packages.txt # System packages present in prod container
+├── pes_frontend_packages.txt # Packages used by react on FE
+├── pes_manifest.txt # contains manifest for Packge Evolution Service
+└── README.md
+```
+
+## How to refresh manifest
+
+1. Refresh pes_container_packages.txt
+    - Go to production pod in openshift and run `rpm -qa`
+2. Refresh pes_backend_packages.txt
+    - Go to production pod in openshift and run `pip freeze`
+    - Note: When refreshing backend packages, remove kernel entry if needed.
```

Comment:
I think this needs to be move to 1. right? since keernel package should be in `rpm -qa`

Reply:
Yep, changed